nrf_security/src/psa_crypto_driver_wrappers.c

```diff
@@ -688,13 +684,14 @@ psa_status_t psa_driver_wrapper_import_key(
                                               key_buffer, key_buffer_size,
                                               key_buffer_length, bits ) );
 #endif /* !MBEDTLS_PSA_BUILTIN_HAS_KEY_TYPE */
-#if defined(PSA_CRYPTO_ACCELERATOR_DRIVER_PRESENT)
-#endif /* PSA_CRYPTO_ACCELERATOR_DRIVER_PRESENT */
+            break;
```

Comment:
This looks like a mistake, break after return.

Reply:
It's hard to trace the valid return code here, but from I can tell we would return PSA_ERROR_CORRUPTION_DETECTED, isn't it more correct to return PSA_ERROR_NOT_SUPPORTED instead?

Reply:
If everything is ifdeffed out here I mean.

Reply:
```
#if defined(MBEDTLS_PSA_BUILTIN_HAS_KEY_TYPE)
            return( psa_import_key_into_slot( attributes,
                                              data, data_length,
                                              key_buffer, key_buffer_size,
                                              key_buffer_length, bits ) );
#else
            return PSA_ERROR_NOT_SUPPORTED;
#endif /* !MBEDTLS_PSA_BUILTIN_HAS_KEY_TYPE */
```

Based on the other functions this is consistent, and I believe the right thing to do also.
I think `psa_driver_wrapper_export_public_key` should also be updated to follow this pattern.

Reply:
IMHO the root of the bug is the unnecessary complexity from doing

```
switch(a) {
  case b:
   default:
            return( PSA_ERROR_INVALID_ARGUMENT );
}
```

instead of just

```
if(a != b) {
  return PSA_ERROR_INVALID_ARGUMENT;
}
```

Reply:
switch-cases with a single case should be avoided, as that is what an if-statement is.

Reply:
**https://github.com/nrfconnect/sdk-nrfxlib/pull/883/files#diff-8838a2c3e7a631a1a8115f848672dfdad351e3769fd43502620a44ec6b964844R815**

here status is initialized with INVALID_ARGUMENT instead of CORRUPT, is it intentional to behave differently in these two functions?